src/javascripts/main.js

```diff
@@ -0,0 +1,71 @@
+$(function () {
```

Comment:
We might want to use the [GOVUK name spacing and module structure](https://github.com/alphagov/styleguides/blob/master/js.md#modules) here. It would probably require restructuring some of the JS so do we want to skip that now? @36degrees

Reply:
Updated to match the _GOVUK name spacing and module structure_, restructured javascripts.

Reply:
Thanks @alex-ju that looks much better. I would prefer to see more variables defined for class names to avoid repeating them but we can revisit that later.